homework05/task3/Program.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+
+namespace task3
+{
+    class Program
```

Comment:
this solution is workable. Also you can do reverse without using a new temp array. 
And this is a real question on an interview - how can you implement a Reverse method without using a new temp array.